docs/sources/migration-guide/migrating-from-single-zone-with-helm.md

```diff
@@ -0,0 +1,719 @@
+         max_global_series_per_user: Z # <-- double it
+   ```
+
+1. Create a new empty YAML file called `migrate.yaml`.
+
+1. Start the migration.
+
+   Copy the following into the `migrate.yaml` file:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         replicas: 0
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step1"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   In this step new zone-aware StatefulSets are created - but no new pods are started yet. The parameter `ingester.ring.zone_awareness_enabled: true` is set in the Mimir configuration via the `mimir.config` value. The flag `-ingester.ring.zone-awareness-enabled=false` is set on distributors, rulers and queriers. The flags `-blocks-storage.tsdb.flush-blocks-on-shutdown` and `-ingester.ring.unregister-on-shutdown` are set to true for the ingesters.
+
+1. Wait for all Mimir components to restart and be ready.
+
+1. Add zone-aware ingester replicas, maximum 21 at a time.
+
+   Explanation: while new ingesters are being added, some series will start to be written to new ingesters, however the series will also exist on old ingesters, thus the series will count twice towards limits. Adding only 21 replicas at a time reduces the number of series affected and thus the likelihood of beaching maximum series limits.
+
+   1. Replace the contents of the `migrate.yaml` file with:
+
+      ```yaml
+      ingester:
+        zoneAwareReplication:
+          enabled: true
+          migration:
+            enabled: true
+            replicas: <N>
+
+      rollout_operator:
+        enabled: true
+      ```
+
+      [//]: # "ingester-step2"
+
+      > **Note**: replace `<N>` with the number of replicas in each step until `<N>` reaches the same number as in `ingester.replicas`, do not increase `<N>` with more than 21 in each step.
+
+   1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   1. Once the new ingesters are started and are ready, wait at least 3 hours.
+
+      The 3 hours is calculated from `blocks_storage.tsdb.block_ranges_period` + `blocks_storage.tsdb.head_compaction_idle_timeout` Grafana Mimir parameters to give enough time for ingesters to remove stale series from memory. Stale series will be there due to series being moved between ingesters.
+
+   1. If the current `<N>` above in `ingester.zoneAwareReplication.migration.replicas` is less than `ingester.replicas`, go back and increase `<N>` with at most 21 and repeat these four steps.
+
+1. If you are using [shuffle sharding]({{< relref "../operators-guide/configure/configuring-shuffle-sharding" >}}), it must be turned off on the read path at this point.
+
+   1. Update your configuration with these values and keep them until otherwise instructed.
```

Comment:
should this go into `migrate.yaml`? Because if yes, then we later instruct them to replace its contents with other things.